### PR TITLE
added backend mod for startdate +1

### PR DESCRIPTION
**Pull request by bashiir**

### CODE REVIEW

File: BACKEND/userAuth/src/main/java/com/apptime/auth/service/TaskManagerService.java [hunk 1]
```diff
@@ -197,10 +201,14 @@ public TaskState complete(long taskId){
 
 	public Set<Task> getTasksStartedLaterThan(Date start, String name) {
 		String pattern = "yyyy-MM-dd";
+		Calendar c = Calendar.getInstance();
+		c.setTime(start);
+		c.add(Calendar.DATE, 1);
 		SimpleDateFormat simpleDateFormat = new SimpleDateFormat(pattern);
-		java.sql.Date qdate = new java.sql.Date(start.getTime());
+		Date sDate = new Date(start.getTime());
```
Comment (smthwas) [inline BACKEND/userAuth/src/main/java/com/apptime/auth/service/TaskManagerService.java:208]: Looks like it is unnecessary to create this instance: we can use `start` directly

File: BACKEND/userAuth/src/main/java/com/apptime/auth/repository/TaskRepository.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 	Task findByUserNameAndState(String userName, TaskState state);
 
 	Task deleteById(long id);
-
-	@Query("select u from Task u where u.scheduledstart >= :start and u.userName = :name")
-    Set<Task> getTasksStartedLaterThan( Date start,String name);
+	//@Query( value = "SELECT * FROM   task WHERE  (scheduledstart AT TIME ZONE 'CDT')::date = :start and username = :name", nativeQuery = true)
```
Comment (smthwas) [inline BACKEND/userAuth/src/main/java/com/apptime/auth/repository/TaskRepository.java:26]: We should remove this line.

File: BACKEND/userAuth/src/main/java/com/apptime/auth/service/TaskManagerService.java (hunk 1 above)
Comment (bashiir) [reply BACKEND/userAuth/src/main/java/com/apptime/auth/service/TaskManagerService.java:208]: good catch removed.

File: BACKEND/userAuth/src/main/java/com/apptime/auth/repository/TaskRepository.java (hunk 2 above)
Comment (bashiir) [reply BACKEND/userAuth/src/main/java/com/apptime/auth/repository/TaskRepository.java:26]: ops! forgot to remove this commented code.

**Pull request merged**